templates/citizen_action_manager/print_participants.html.twig

```diff
@@ -0,0 +1,6 @@
+{% extends 'base_print.html.twig' %}
+
+{% block content %}
+    <p class="text--bold">Liste des participants</p>
+    {% include 'citizen_action/_participants_list.html.twig' with { participants: participants, is_host: true } only %}
```

Comment:
useless `participants: participants`

Reply:
Here it's useful, theses variables are not introduced in parent template.